app/ide-desktop/lib/content/src/index.ts

```diff
@@ -955,6 +955,7 @@ async function runEntryPoint(config: Config) {
             show_debug_screen(wasm, "Unknown entry point '" + entryTarget + "'. ")
         }
     } else {
+        loader.destroy()
```

Comment:
Could this not be moved to before the `if` block to avoid it being there twice in different branches?

Reply:
Not really, because this code is also used by `main_entry_point`, and in IDE loader is removed *after* initialization. If we move this line before the if block - it would change the visible behavior of the loader in the IDE.

Reply:
@vitvakatu this is very important information - add it as a comment to the code please

Reply:
Done